docs/website/src/site/styles/homepage/call-to-action.scss

```diff
@@ -0,0 +1,311 @@
+@import './../breakpoints';
+
+$transition-timing: 200ms;
+
+.ds-call-to-action {
+  background: var(--cta-background);
+  border-radius: 1.2rem;
+  direction: ltr;
+  display: grid;
+  gap: 4rem;
+  grid-template-columns: none;
+  grid-template-rows: 1fr auto;
+  justify-items: center;
+  margin: 3.2rem 0;
+  overflow: hidden;
+  padding: 4rem 3.2rem;
+
+  @media screen and (min-width: $md-lg-breakpoint) {
+    direction: rtl;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: none;
+    padding: 4rem;
+  }
+
+  @media screen and (min-width: $lg-breakpoint) {
+    direction: ltr;
+    grid-template-columns: none;
+    grid-template-rows: 1fr auto;
+    justify-items: center;
+    margin: 3.2rem 0;
+    padding: 4rem 3.2rem;
+  }
+
+  @media screen and (min-width: $xl-breakpoint) {
+    direction: rtl;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: none;
+    padding: 4rem;
+  }
+
+  p:not(.ds-call-to-action__main-text):not(.ds-call-to-action__heading) {
+    color: var(--cta-link-color);
+    font-size: 2rem;
+    line-height: 2.4rem;
+    margin-bottom: 1.6rem;
+  }
+
+  svg {
+    height: 30rem;
+    overflow: visible;
+  }
+}
+
+.ds-call-to-action__description {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-self: left;
+
+  @media screen and (min-width: $md-lg-breakpoint) {
+    direction: ltr;
+  }
+}
+
+.ds-call-to-action__eye-brow {
+  animation: scaleAnimationRotate 4s infinite;
+  transform-box: fill-box;
+  transform-origin: center;
+
+  @media screen and (prefers-reduced-motion: reduce) {
+    /* stylelint-disable-next-line max-nesting-depth */
+    animation: none;
+  }
+}
+
+.ds-call-to-action__eye {
+  animation: scaleAnimationEye 5s infinite, blink 5s infinite;
+  backface-visibility: hidden;
+  perspective: 100rem;
+  transform: translate3d(0, 0, 0);
+  transform-box: fill-box;
+  transform-origin: center;
+
+  @media screen and (prefers-reduced-motion: reduce) {
+    /* stylelint-disable-next-line max-nesting-depth */
+    animation: none;
+  }
+}
+
+.ds-call-to-action__smile {
+  animation: smileAnimation 3s infinite;
+  animation-delay: $transition-timing;
+  backface-visibility: hidden;
+  transform: translate3d(0, 0, 0);
+  transform-box: fill-box;
+  transform-origin: center;
+
+  @media screen and (prefers-reduced-motion: reduce) {
```

Comment:
you can also group all the animations together in `@media (prefers-reduced-motion: no-preference)` instead of overwriting it.